### PR TITLE
Minor Strange Reagent Tweaks

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Just a few minor tweaks to SR--brings it closer to Goon's and makes it far less annoying to use:

- Strange Reagent can work with a touch based reaction, but only 25% of the time
- As long as you're below the damage threshhold to gib (and aren't suiciding/uncloneable/hellbound), the person will revive, no matter what, ghost or not.
  - This means you can use it on NPCs like monkies
- When someone revives with Strange reagent, they are forced back into the body

Other:
- When you force eject a cloner, it ejects the unattched organs.
- Gibs will have the proper dna of the person who went splat in the cloner

:cl: Fox McCloud
tweak: Strange Reagent will work 25% of the time with TOUCH based reactions
tweak: Strange reagent will revive whether there's a ghost or not (can use SR on clientless mobs now, like monkies)
tweak: SR puts the occupant back in their body automatically
tweak: Cloners that eject their messes will now eject the unattached organs
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/medicine.dm [hunk 1]
```diff
@@ -715,8 +709,8 @@
 									O.germ_level = INFECTION_LEVEL_THREE
 						H.update_body()
 
+					M.grab_ghost()
 					M.update_revive()
-					M.stat = UNCONSCIOUS
```
Comment (Fox-McCloud) [inline code/modules/reagents/chemistry/reagents/medicine.dm]: Not needed, `update_revive` handle all the `stat` setting

**Comment by Aurorablade** [comment]
"When you force eject on a cloner, it eject the unattched organs."....Stop channeling me...

**Comment by KasparoVy** [comment]
What a wonderful mess we've made.

**Comment by Mitchs98** [comment]
Thank god. No more cramming 6 SR pills down someones' throat on the off-chance they may rejoin their body. If they really don't wanna live again they can just ghost.

**Comment by KasparoVy** [comment]
I can already see people suddenly coming back to life mid-deadchat-salt and the first sentence out of their revived face is something totally wack

**Comment by TDSSS** [comment]
>When someone revives with Strange reagent, they are forced back into the body

I can already see all the revived people that wanted to try their luck getting terror spiders or whatever beelining for cryo, lol.

**Comment by SkeletalElite** [comment]
I don't really like SR forcing you back into the body, perhaps do what TG does and have a ghost verb that you can use if you don't want to be revived that will make you not able to re enter your body again but also keep your respawnability. It's really annoying if someone wants to stay dead but get SRd and are only left with the choice of ghosting which removes your respawnability and often times its either that or 20 minutes of unconsciousness while doctors fix everything on you.

**Pull request merged**